project/project.go

```diff
@@ -40,7 +43,11 @@ func RelPPath(p string) string {
 }
 
 func BuildOut() string {
-	if os.Getenv("BUILD_OUT") != "" {
+	if *copyToRoot || Getenv("COPYTOROOT") == "true" {
```

Comment:
to be honest I'm really not a massive fan either. the Getenv is only there because we can set some defaults for things in the `bldy.cfg` file and distribute it. But you are right there is no real reason for it to be there. I'll get rid of it.